### PR TITLE
Koala event fix

### PR DESCRIPTION
**Pull request by Scollaco**
# What
Change the place where we send `current_variants` property with Koala events.

# Why
Because we were sending `current_variants` with `Viewed Discovery` Koala event, the property was being renamed to `discover_current_variants`, what doesn't exist on the backend.

### CODE REVIEW

File: Library/Koala/KoalaTests.swift [hunk 1]
```diff
@@ -356,7 +356,7 @@ final class KoalaTests: TestCase {
     XCTAssertNil(properties["discover_staff_picks"])
     XCTAssertNil(properties["discover_starred"])
     XCTAssertNil(properties["discover_term"])
-    XCTAssertEqual(false, properties["discover_everything"] as? Bool)
+    XCTAssertEqual(true, properties["discover_everything"] as? Bool)
```
Comment (justinswart) [inline Library/Koala/KoalaTests.swift]: This test seems unrelated, just curious why the boolean flipped?

Comment (Scollaco) [reply Library/Koala/KoalaTests.swift]: The value `true`  means that the array of "discovery properties"  is empty in that context. The value was `false` because we adding the `current_variants` property  to the array, but now that we moved the property to another place, the array is empty again (in this test).
